### PR TITLE
chore(starters): add gatsby-starter-leaflet

### PR DESCRIPTION
**Pull request by colbyfayock**
## Description
* Adds [gatsby-starter-leaflet](https://github.com/colbyfayock/gatsby-starter-leaflet) to showcase.
* Adds new category Mapping:Leaflet - the goal is to both provide the top level category of Mapping to encourage more map submissions with a subset of Leaflet, as there are other popular Mapping frameworks available.

### CODE REVIEW

File: docs/categories.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ starter:
   - Language:Other
   - Language:TypeScript
   - Linting
+  - Mapping:Leaflet
```
Comment (wardpeet) [inline docs/categories.yml]:
```suggestion
```

File: docs/starters.yml [hunk 2]
```diff
@@ -4061,3 +4061,17 @@
     - Google Analytics
     - Simple design
     - Markdown support
+- url: https://gatsby-starter-leaflet.netlify.com/
+  repo: https://github.com/colbyfayock/gatsby-starter-leaflet
+  description: A Gatsby starter with Leafet!
+  tags:
+    - Landing Page
+    - Linting
+    - Mapping:Leaflet
```
Comment (wardpeet) [inline docs/starters.yml]:
```suggestion
```

**Comment by colbyfayock** [comment]
@wardpeet sure thing, updated it to just `Mapping`

**Comment by wardpeet** [comment]
We don't want to add Mapping either 😛. Sorry about this but we don't want to add to many categories to our list. We also need to remove some that shouldn't be there.

**Comment by colbyfayock** [comment]
fair enough, though I 🤞maybe it'll appear some day. it's an increasingly growing space with a lot of interesting technical work. we're building https://www.element84.com/dr/ on Gatsby for instance :)

removed from both files

**Pull request merged**